src/formatDate.js

```diff
@@ -50,7 +50,23 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const obj = {};
+  const result = [];
+  const str = date.split(fromFormat[3]);
```

Comment:
you have an array after split, why do you call it `str`?

Reply:
changed